File: RxJava2Study/src/main/java/com/baiiu/rxjava2study/MainActivity.java
package com.baiiu.rxjava2study;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;


public class MainActivity extends AppCompatActivity {

    @Override protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        getSupportFragmentManager().beginTransaction()
                .add(android.R.id.content, getFragment(), "operator")
                .commit();

    }

    public Fragment getFragment() {

        return new CreateFragment();

    }
}

File: plugin/pluginApplication/src/main/java/com/baiiu/pluginapplication/MainActivity.kt
package com.baiiu.pluginapplication

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        one()
    }

    fun onClick(view: View) {
        startActivity(Intent(this, SecondActivity::class.java))
    }

    fun one() {
        two()
    }

    fun two() {
        three()
    }

    fun three() {

    }

}
